.changeset/early-spiders-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@ai-sdk/google-vertex': major
+'@ai-sdk/google': patch
+---
+
+feat (provider/google-vertex): Add support for edge runtime.
```

Comment:
As a major version bump, do we link to more info or provide more detail in the message?

Reply:
Maybe add that it was a rewrite of the vertex provider. That is enough justification for a major imo.